service/damagePattern.py

```diff
@@ -36,9 +37,7 @@ def getInstance(cls):
     def __init__(self):
         uniform = eos.db.getDamagePattern("Uniform")
         if uniform is None:
-            uniform = eos.types.DamagePattern(25, 25, 25, 25)
-            uniform.name = "Uniform"
-            eos.db.save(uniform)
+            loadDefaultDatabaseValues.defaultDatabaseValues.importRequiredDefaults()
```

Comment:
Once the import is fixed, this can be simplified to `DefaultDatabaseValues.importRequiredDefaults()`